test/integration/validation/chain_validator_storage_test.cpp

```diff
@@ -142,8 +151,8 @@ namespace iroha {
   /**
    * @given initialized storage with 4 peers
    * block 1 - initial block with 4 peers
-   * block 2 - signed by supermajority of ledger peers
-   * block 3 - signed by supermajority of ledger peers
+   * block 2 - signed by all ledger peers
```

Comment:
What are the reasons for these changes?

Reply:
I want to remove the supermajority logic related assumptions from this test. So previously we assumed that some amount of peers are supermajority, and now we remove that assumption because all peers voting unanimously are supermajority (almost) regardless of its model.